lib/tasks/dynflow.rake

```diff
@@ -0,0 +1,5 @@
+namespace :dynflow do
```

Comment:
In development, to have a separate executor from the main process (https://github.com/theforeman/foreman-tasks/#production-mode)